Bookish/Client/Pages/History/History.razor

```diff
@@ -0,0 +1,4 @@
+﻿@page "/Account/History"
+
+<h1>My Activity</h1>
+<Bookish.Client.Shared.Components.PostList Posts="@Posts"></Bookish.Client.Shared.Components.PostList>
```

Comment:
Comments? This will be in order from newest to oldest. There is an endpoint now. /api/history

Reply:
It will be a list of ilistitem